parse/Parse.h

```diff
@@ -48,12 +48,17 @@ namespace parse {
 
     FO_PARSE_API bool ship_hulls(std::map<std::string, HullType*>& hulls);
 
+    /** Parse all ship designs in directory \p path, store them in \p designs. If a file exists
+        called ShipDesignManifest.focs.txt, parse it and store the order in \p ordering. */
```

Comment:
Does `ShipDesignManifest.focs.txt` do anything besides determine the order in which the designs are displayed? If not, I think another name would be clearer about its purpose...